cmd/clusterctl/cmd/move.go

```diff
@@ -29,6 +29,9 @@ type moveOptions struct {
 	toKubeconfig          string
 	toKubeconfigContext   string
 	namespace             string
+	directory             string
+	restoreFromFolder     bool
+	backupToFolder        bool
```

Comment:
Let's avoid the usage of backup and restore everywhere
```suggestion
	fromDirectory     string
	toDirectory        string
```